lab-teddy/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "lab-2",
+  "version": "1.0.0",
```

Comment:
so you don't actually have mocha installed as a dependacy. I would npm i -D mocha